packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,381 @@ components:
+              properties:
+                url:
+                  type: string
+                  description: URL of the asset to import
+                  example: https://cdn.livepeer.com/ABC123/filename.mp4
+            export:
+              type: object
+              additionalProperties: false
+              description: Parameters for the export task
+              properties:
+                kind:
+                  type: string
+                  description: Type of the export
+                  example: arweave|ipfs|s3
+                s3:
+                  type: object
+                  additionalProperties: false
+                  description: Parameters for the S3 export
+                  properties:
+                    destination:
+                      type: string
+                      description: Destination of the export
+                      example: s3://bucket/path/to/file.mp4
+                    key:
+                      type: string
+                      description: Key of the export
+                      example: ABC123
+                    secret:
+                      type: string
+                      description: Secret of the export
+                      example: ABC123
+            transcode:
+              type: object
+              additionalProperties: false
+              description: Parameters for the transcode task
+              properties:
+                assetId:
+                  type: string
+                  description: ID of the asset to transcode
+                  example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+                preset:
+                  type: string
+                  description: Preset to use for the transcode
+                  example: 1080p
+                container:
+                  type: string
+                  description: Container to use for the transcode
+                  example: mp4
+                audio:
+                  type: object
+                  additionalProperties: false
+                  description: Audio parameters for the transcode
+                  properties:
+                    codec:
+                      type: string
+                      description: Codec to use for the audio
+                      example: aac
+                    bitrate:
+                      type: number
+                      description: Bitrate of the audio
+                      example: 128000
+                    channels:
+                      type: number
+                      description: Amount of audio channels
+                      example: 2
+                    sampleRate:
+                      type: number
+                      description: Sample rate of the audio
+                      example: 44100
+                    bitDepth:
+                      type: number
+                      description: Bit depth of the audio
+                      example: 16
+                video:
```

Comment:
I **think** `cli-transcoder` only supports some preset profiles for now.  We don't need to check that now anyway, let's create this with only the import task.